templates/backup_large_repo.service.j2

```diff
@@ -0,0 +1,59 @@
+# Managed by Ansible, please don't edit manually
+
+[Unit]
+Description=borgmatic backup only - because large repo
```

Comment:
The normal/large hack isn't needed any more since Borgmatic added a setting for it:

https://torsion.org/borgmatic/docs/how-to/deal-with-very-large-backups/#check-frequency

Best to just add this to `config.yaml.j2`.